eventmon/monitord/impl.go

```diff
@@ -37,6 +43,8 @@ func newMonitor(keymasterServerHostname string, keymasterServerPortNum uint,
 	webLoginChannel := make(chan string, bufferLength)
 	x509RawCertChannel := make(chan []byte, bufferLength)
 	x509CertChannel := make(chan *x509.Certificate, bufferLength)
+	Teelog := teelogger.New(One, Two)
+	defer One.Close()
```

Comment:
I think this will cause system logging to fail once you return from this function.